src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java

```diff
@@ -184,7 +184,7 @@ private void handleImageException(ServiceException exception, Request request, f
         g.setColor(bgcolor);
         g.fillRect(0, 0, img.getWidth(), img.getHeight());
         
-        if (!"BLANK".equals(exceptionFormat)) { //wms 1.3 only
```

Comment:
AFAIK GeoServer isn't writing text into blank images. The application/vnd.ogc.se_blank WMS 1.1.1 type is simply added to the original implementation for a blank exception.